### ISSUE TITLE
[0.9.5] Vamp keeps deploying state, even when task is already done and started

### ISSUE THREAD

**Issue by tnolet** [magneticio/vamp#1016]
Deploying the following blueprint. The start up time of the songs service is slightly longer than normal due to seed data being inserted into the DB. However, even after it finishes and DCOS recognises the services as up and running, Vamp still keeps it in the deploying state.

![image](https://user-images.githubusercontent.com/3802923/29084645-fa0c4eb0-7c6c-11e7-875a-5b8e51ab11e2.png)

- After some time, Vamp marks the deployment as failed
- Deleting the deployment does not remove the task in DCOS

 the end Vamp marks the deployment as failed

```yaml
---
name: vamp-music-demo
gateways:
  3000: songs/http_port
clusters:
  web:
  songs:
    services:
      breed:
        name: songs:1.0
        deployable: magneticio/vamp-music-songs
        ports:
          http_port: 3004/http
        dependencies:
          db: songs-mongo
        environment_variables:
          MONGO_HOST: $db.host
          MONGO_PORT: $db.ports.mongodb_port
      dialect:
        marathon:
          healthChecks:
            - maxConsecutiveFailures: 0
          docker:
            force_pull_image: true
      scale:
        cpu: 1
        memory: 2GB
        instances: 1
  songs-mongo:
    services:
      name: songs-mongo
      deployable: mongo:latest
      ports:
        mongodb_port: 27017/tcp
```

**Comment by tnolet** [comment]
@mihaiSimu I got it working in the end by doing two things:
1. I drastically shortened the startup time, Although I don't think this was the thing blocking it. This specific container starts a web server on port 3000 and in the background starts filling a database. This means port 3000 is already excepting connections, even when the data is not filled completely.
2. I removed the dialect stanzas. I think this might have been the cause of the issue.
